### PR TITLE
Help command-line options

### LINKED ISSUE #146: command-line options
bin/app should handle --help and errors in command line options

### PR DESCRIPTION
**Pull request by ventilooo**
fix #146 
Signed-off-by: iszalai szalai972@gmail.com

---
This change is [Reviewable](https://reviewable.io/reviews/morriar/missions/157)

### CODE REVIEW

**Review by privat — CHANGES_REQUESTED** [review]
-1 This does not use the existing -h option from popcorn nor the facilities of the opts library.

**Comment by ppepos** [comment]
What @privat said.

**Comment by ventilooo** [comment]
reroll @ppepos @privat

File: src/app.nit [hunk 1]
```diff
@@ -15,6 +15,13 @@
 import api
 
 var opts = new AppOptions.from_args(args)
+
+if opts.opt_help.value == true then
```
Comment (lbajolet) [inline src/app.nit]: No need for the `== true` as value is already a boolean

File: src/app.nit [hunk 2]
```diff
@@ -15,6 +15,13 @@
...
+    print("Usage: app [Options]\nOptions:")
+    opts.usage
+    exit(0)
```
Comment (lbajolet) [inline src/app.nit]: From what I remember, @privat advises against `exit(0)` in favor of a `return 0`

**Review by lbajolet — CHANGES_REQUESTED** [review]
When both my remarks are addressed, I'd say +1 unless someone else votes against

**Comment by ventilooo** [comment]
fix according to @R4PaSs review

**Review by privat — APPROVED** [review]

**Comment by privat** [comment]
ok to test

**Review by lbajolet — APPROVED** [review]

**Pull request merged**